src/models/slices/filters.ts

```diff
@@ -1,8 +1,18 @@
-import { DisplayableFacet, SortBy } from '@yext/answers-core';
+import { DisplayableFacet } from '@yext/answers-core';
 import { SelectableFilter } from '../utils/selectablefilter';
 
+/**
+ * Maintains the current state of facets and filters in the application.
+ */
 export interface FiltersState {
+  /**
+   * The collection of possible static filters that can be applied to the
```

Comment:
Can we also indicate that it maintains which filters are currently applied?

Reply:
Sure! This should be added for both static filters and facets, right?

Reply:
Yep

Reply:
Updated